package.json

```diff
@@ -12,23 +12,25 @@
   "bin": "bin/pipeline.js",
```

Comment:
Perhaps, also update this bin pointer to `bin/orcabus.ts` as default.

Reply:
I think we could remove this bin